src/protocol/ipa_prf/boolean_ops/addition_sequential.rs

```diff
@@ -19,13 +19,12 @@ pub(crate) enum Step {
 /// adds y to x, Output has same length as x (carries and indices of y too large for x are ignored)
```

Comment:
Can you update the documentation here?

The original here didn't use sentences (it should) and didn't describe what it returned.  That meant that you didn't need to update it, but we can do better.